src/history/history-entry-import.dto.ts

```diff
@@ -4,12 +4,18 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
-import { IsString } from 'class-validator';
+import { IsBoolean, IsString } from 'class-validator';
 
-export class HistoryEntryCreationDto {
+export class HistoryEntryImportDto {
   /**
    * ID or Alias of the note
    */
   @IsString()
   note: string;
+  /**
+   * True if the note should be pinned
+   * @example true
+   */
+  @IsBoolean()
+  pinStatus: boolean;
```

Comment:
We also need the `lastVisited` time here as that can be imported from the frontend too. It should not be set to the time of the import by the backend.

Reply:
added